app/screens/edit_post/edit_post.tsx

```diff
@@ -162,7 +162,7 @@ const EditPost = ({componentId, maxPostSize, post, closeButtonId, hasFilesAttach
         toggleSaveButton(post.message !== message);
     }, [intl, maxPostSize, toggleSaveButton]);
 
-    const handleUIUpdates = useCallback((res) => {
+    const handleUIUpdates = useCallback((res: {error?: unknown}) => {
```

Comment:
If res is always defined, there is no need for the optional chaining (`?.`) in the next line. If not, it should be optional here.